streams/ni.m3u

```diff
@@ -3,8 +3,8 @@
 http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.dailymotion.com/canal6nicaraguaoficial
 #EXTINF:-1 tvg-id="Canal9.ni" status="timeout",Canal 9 (480p) [Timeout]
 https://cootv.cootel.com.ni/streams/d/SSLCH09-HD-CVS/playlist.m3u8
-#EXTINF:-1 tvg-id="Canal10.ni" status="timeout",Canal 10 (576p) [Timeout]
-https://cootv.cootel.com.ni/streams/d/SSLCH10-SD-ZUK/playlist.m3u8
+#EXTINF:-1 tvg-id="Canal10.ni" status="timeout",Canal 10 (480p) [Geo-blocked]
+http://free.fullspeed.tv/iptv-query?streaming-ip=https://www.canal10.com.ni/envivo/
```

Comment:
The link no longer works

<img width="757" alt="image" src="https://user-images.githubusercontent.com/7253922/155676264-d2ad8800-3e44-412e-b9de-372fb73ad6f0.png">

Reply:
The stream is only available for news. From 5:00 to 9:00, then from 13:00 to 14:00, and then from 19:00 to 20:00. [UTC-06:00]. The rest of the time is restricted.

Reply:
In this case it should be marked as `[Not 24/7]`